app/ents/employee/endpoints.py

```diff
@@ -70,19 +73,18 @@ def login_employee():
             code=HTTPStatus.OK,
             token=security.create_token(employee),
         )
-
+    
     return error_response(error=EmployeeDoesNotExist.msg, code=HTTPStatus.BAD_REQUEST)
 
 
 @bp.after_request
-# @jwt_required
 def refresh_expiring_jwts(success_response):
```

Comment:
Can some of the logic here be extracted to app.core.security?

Reply:
okay I will encaspulate that in there

Reply:
kindly check latest commit to see if it looks good